xml/abstract_commands.xml

```diff
@@ -227,12 +263,16 @@
         </field>
         <field name="0" bits="18:17" />
         <field name="write" bits="16">
-            0: Copy data from the memory location specified in {\tt arg1} into
+            <value v="0" name="arg">
+            Copy data from the memory location specified in {\tt arg1} into
             the low bits of {\tt arg0}. Any remaining bits of {\tt arg0} now
             have an undefined value.
```

Comment:
I just noticed but should this be `\unspecified` instead of undefined?

Reply:
Done.